src/app/decks/Decks.js

```diff
@@ -35,7 +35,7 @@ class Decks extends Component {
     if (searchParams.beta) {
```

Comment:
can you also explain how this works?

Reply:
Yeah it's been our mechanism to test staged topics. It checks the url for `?beta=true` and fetches all decks instead of fetching only the published decks.

This can change with the collections page actually.

Reply:
Gotcha, I just couldn't make sense of it when I was reviewing. Makes sense now. Up to you if you want to remove it 👍